adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -1369,6 +1462,34 @@ abstract class MultisampleAvroGenomicRDD[T <% IndexedRecord: Manifest, U <: Prod
    */
   val headerLines: Seq[VCFHeaderLine]
```

Comment:
This illustrates something I didn't notice before, the only subclass of `MultisampleAvroGenomicRDD` is `GenotypeRDD`.

`VariantContextRDD` extends `MultisampleGenomicRDD` and adds its own `headerLines` and `VariantRDD` extends `AvroGenomicRDD` and adds its own `headerLines`.

If `MultisampleAvroGenomicRDD` was intended to be a superclass for other data types, an association to VCF headers doesn't really make sense.

To address this, we could remove `MultisampleAvroGenomicRDD` and move `headerLines` to `GenotypeRDD`, or perhaps create something new for VCF `headerLines` that all of `VariantContextRDD`, `VariantRDD`, and `GenotypeRDD` could use.